src/libraries/System.Private.CoreLib/src/System/SpanHelpers.cs

```diff
@@ -407,6 +408,26 @@ public static unsafe void ClearWithReferences(ref IntPtr ip, nuint pointerSizeLe
             ip = default;
         }
 
+        // This can be replaced once https://github.com/dotnet/runtime/issues/63331 is implemented.
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        private static Vector128<byte> SimdShuffle(Vector128<byte> left, Vector128<byte> indices)
```

Comment:
I think we already have this exact helper somewhere so you can just move it to `Vector128` as an internal API

Reply:
Done